### PR TITLE
Release 11.2.0 rc1

### PR DESCRIPTION
**Pull request by pascalwengerter**
This branch is just meant to create the RC1, don't merge

### CODE REVIEW

File: src/components/molecules/OcTable/OcTable.spec.js [hunk 1]
```diff
@@ -195,15 +195,30 @@ describe("OcTable", () => {
...
+  it("adds data-item-id for rows", () => {
     const wrapper = shallowMount(Table, {
       propsData: {
         fields,
         data,
         highlighted: [],
       },
     })
-    expect(wrapper.html().indexOf("data-file-id")).toBeGreaterThan(-1)
+    console.log(wrapper.html())
```
Comment (kulmann) [inline src/components/molecules/OcTable/OcTable.spec.js:206]: dev leftover?

Comment (pascalwengerter) [reply src/components/molecules/OcTable/OcTable.spec.js:206]: Ah yeah I added this in my review for the original PR by @fschade, not relevant in the context of this one as it is only supposed to run against the web CI for a quick check

**Comment by pascalwengerter** [comment]
Closing this PR as the RC release is done

**Comment by pascalwengerter** [comment]
Green CI in https://github.com/owncloud/web/pull/6006, closing this one to review and hopefully officially release #1766

**Pull request closed without merging**